composer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "riesenia/cakephp-routing",
+    "description": "cakephp attribute routing",
+    "homepage": "https://cakephp.org",
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "cakephp/cakephp": "^5.0.1",
+        "php": "^8.1",
+        "mixerapi/core": "^2.0"
```

Comment:
why do we need mixerapi/core?

Reply:
i use the namespaceUtility instead of the glob for getting controllers
in RoutesCommand:
` NamespaceUtility::findClasses($controllerNs . 'Controller'))`

Reply:
ok... but if you check NamespaceUtility::findClasses method, you'll find out, you need `thecodingmachine/class-explorer` package